src/components/Container/src/Container.vue

```diff
@@ -10,22 +10,29 @@
 	>
 		<header :class="$s.Header">
 			<div
-				v-if="label"
+				v-if="hasLabel"
 				:class="$s.Label"
 			>
-				{{ label }}
-
+				<!-- @slot container label -->
+				<slot name="label">
+					{{ label }}
+				</slot>
 				<div
-					v-if="sublabel"
+					v-if="hasSublabel"
 					:class="$s.Sublabel"
 				>
-					{{ sublabel }}
+					<!-- @slot container sublabel -->
+					<slot name="sublabel">
+						{{ sublabel }}
+					</slot>
 				</div>
 			</div>
 
 			<div :class="$s.RequirementLabel">
```

Comment:
Does this need a v-if check like label and sublabel?

Reply:
hmm, it doesn't really _need_ it but I've added one anyway for consistency's sake :)